Apps/Admin/Server/Admin.Server.csproj

```diff
@@ -32,4 +32,8 @@
       <GlobalPropertiesToRemove></GlobalPropertiesToRemove>
     </ProjectReference>
   </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="Properties\ServiceDependencies\hgadmin - Web Deploy\profile.arm.json" />
+  </ItemGroup>
```

Comment:
This change to the .csproj can be reverted.

Reply:
Reverted.